pure-eth-connector/src/connector.rs

```diff
@@ -0,0 +1,276 @@
+        let required_deposit =
+            Balance::from(current_storage - initial_storage) * sdk::STORAGE_PRICE_PER_BYTE;
+        self.save_contract();
+        attached_deposit - required_deposit
+    }
+
+    fn mint(&mut self, owner_id: AccountId, amount: Balance) {
+        #[cfg(feature = "log")]
+        sdk::log(format!("Mint {:?} tokens for: {:?}", amount, owner_id));
+
+        let owner_id_key: &str = owner_id.as_ref();
+        if self.contract.token.accounts.get(owner_id_key).is_none() {
+            // TODO: NEP-145 Account Storage implementation fee
+            // It spent additional account amount for storage
+            self.contract.token.accounts.insert(owner_id.clone(), 0);
+        }
+        self.contract.token.internal_deposit(&owner_id, amount);
+        self.save_contract();
+        #[cfg(feature = "log")]
+        sdk::log("Mint success".into());
+    }
+
+    fn burn(&mut self, owner_id: AccountId, amount: Balance) {
+        #[cfg(feature = "log")]
+        sdk::log(format!("Burn {:?} tokens for: {:?}", amount, owner_id));
+        self.contract.token.internal_withdraw(&owner_id, amount);
+    }
+
+    pub fn withdraw(&mut self) {
+        #[cfg(feature = "log")]
+        sdk::log("Start withdraw".into());
+        let args: WithdrawCallArgs = serde_json::from_slice(&sdk::read_input()[..]).unwrap();
+        let recipient_address = validate_eth_address(args.recipient_id);
+        // Burn tokens to recipient
+        self.burn(sdk::predecessor_account_id(), args.amount);
+        let res = WithdrawResult {
+            recipient_id: recipient_address,
+            amount: args.amount,
+            eth_custodian_address: self.contract.eth_custodian_address,
+        }
+        .try_to_vec()
+        .unwrap();
+        sdk::value_return(&res[..]);
+    }
+
+    pub fn ft_total_supply(&self) {
+        let total_supply = self.contract.token.ft_total_supply();
+        sdk::value_return(&total_supply.to_be_bytes());
+        #[cfg(feature = "log")]
+        sdk::log(format!("Total supply: {}", total_supply));
+    }
+
+    pub fn ft_balance_of(&self) {
+        let args: BalanceOfCallArgs = serde_json::from_slice(&sdk::read_input()[..]).unwrap();
+        let balance = self.contract.token.ft_balance_of(args.account_id.clone());
+        sdk::value_return(&balance.to_be_bytes());
+        #[cfg(feature = "log")]
+        sdk::log(format!("Balance [{}]: {}", args.account_id, balance));
+    }
+
+    pub fn ft_transfer(&mut self) {
+        let args: TransferCallArgs = serde_json::from_slice(&sdk::read_input()[..]).unwrap();
+
+        self.contract
+            .token
+            .ft_transfer(args.receiver_id.clone(), args.amount, args.memo.clone());
+        self.save_contract();
+        #[cfg(feature = "log")]
+        sdk::log(format!(
+            "Transfer to {} amount {} with memo {:?} success",
+            args.receiver_id, args.amount, args.memo
+        ));
```

Comment:
Change this to:
```rust
        sdk::log(format!(
            "Transfer amount {} to {} success with memo: {:?}",
            args.amount, args.receiver_id, args.memo
        ));
```
Since `memo` is an `Option` it should be tagged on following the grammatical english and `:`.